lld/ELF/Thunks.cpp

```diff
@@ -69,6 +69,7 @@ class AArch64Thunk : public Thunk {
 private:
   bool mayUseShortThunk = true;
   virtual void writeLong(uint8_t *buf) = 0;
+  virtual void addLongMapSyms() {}
```

Comment:
Perhaps add some comment, something like: mapping symbols with non-zero offsets may become needed when mayUseShortThunk becomes false later.

Reply:
Thanks for the review, I've updated the comment. I'll also update the description as suggested below when merging.